backend/app.ts

```diff
@@ -594,21 +611,14 @@ io.on(M.connectionMsg, function (socket: any): void {
   // Utility functions.
   ////////////////////////////////////
 
-  function isValidEloId(eloId: string | null | undefined): boolean {
-    return (
-      eloId !== null &&
-      eloId !== undefined &&
-      eloId !== "" &&
-      eloId !== "undefined" &&
-      eloId.length >= 4 &&
-      eloId !== "elo_"
-    );
+  function isValidIdToken(idToken: string | null | undefined): boolean {
+    return idToken !== null && idToken !== undefined && idToken !== "undefined";
```

Comment:
would it be safer to check that it's not the empty string?

Reply:
done